### PR TITLE
OTE branch substitution

### PR DESCRIPTION
**Pull request by kprokofi**

### CODE REVIEW

**Comment by kprokofi** [comment]
@sovrasov , please look at 3 last commits (I added scale parameter for epochs and learning rate)

File: torchreid/integration/sc/train_task.py [hunk 1]
```diff
@@ -88,6 +88,10 @@ def train(self, dataset: DatasetEntity, output_model: ModelEntity,
             update_progress_callback = train_parameters.update_progress
         else:
             update_progress_callback = default_progress_callback
+
+        if self._cfg.type == 'multilabel':
```
Comment (sovrasov) [inline torchreid/integration/sc/train_task.py]: `if self._multilabel:`

File: scripts/default_config.py [hunk 2]
```diff
@@ -116,6 +116,7 @@ def get_default_config():
     cfg.train.lr = 0.0003
     cfg.train.weight_decay = 5e-4
     cfg.train.max_epoch = 60
+    cfg.train.epoch_scale = 3.
```
Comment (sovrasov) [inline scripts/default_config.py]: This is a task-specific parameter, which doesn't affect train via main.py. Logically it doesn't belong to d-o-r yaml config. We can put the new params to ConfigurableParameters, however, they are not convenient to modify. I propose adding this stuff to yaml, into a standalone group like `sc_integration`, subgroup `multilabel`.

Comment (kprokofi) [reply scripts/default_config.py]: done

File: torchreid/integration/sc/train_task.py (hunk 1 above)
Comment (kprokofi) [reply torchreid/integration/sc/train_task.py]: done

**Comment by kprokofi** [comment]
Have checked that `ote_sample` is runnable with mobilenetv3_large_1x and EfficientNetv2_s

**Comment by kprokofi** [comment]
@sovrasov ,  How the tests are going? Are we ready to merge?

**Comment by sovrasov** [comment]
Not yet, tests are in progress

**Comment by sovrasov** [comment]
Builds are ok, we can finally merge it!

**Review by sovrasov — APPROVED** [review]
